js/source/cluster_source.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var util = require('../util/util');
+var GeojsonSource = require('./geojson_source');
+var webworkify = require('webworkify');
+
+module.exports = ClusterSource;
+module.exports.worker = URL.createObjectURL(webworkify(require('./geojson_worker_supercluster'), {bare: true}));
+
+function ClusterSource (options) {
```

Comment:
It feels funny that the `GeoJSONSource` is named by the type of data it processes while `ClusterSource` is named by a transformation it performs on the data it processes. Can you think of any way to make the source naming more consistent? `ClusteredGeoJSONSource`?